analyzers/its/sources/Net5/Net5/S1541.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Net5
+{
+    public class S1541
+    {
+        public int Property
+        {
+            get
+            {
+                return 0;
+            }
+            init
+            {
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
+                if (false) { }
```

Comment:
Why so many? Shouldn't just 2 or 3 of them do the same trick?

Reply:
Since the default minimum value is 10. So you need to launch this rule with a custom parameter to make it work here.